morpheus/CMakeLists.txt

```diff
@@ -20,13 +20,8 @@ option(MORPHEUS_PYTHON_INPLACE_BUILD
 option(MORPHEUS_BUILD_PYTHON_STUBS
         "Whether or not to generate stubs for python files." ON)
 
-set(Python3_FIND_VIRTUALENV "FIRST")
-set(Python3_FIND_STRATEGY "LOCATION")
-find_package(Python3 REQUIRED COMPONENTS Development Interpreter NumPy)
-
-# Check if SKlearn is installed
-include(cmake/utils/sklearn_check.cmake)
-include(cmake/utils/python_module_tools.cmake)
+# TODO(Devin): Not currently using this, just adding to make module and lib creation
+morpheus_utils_create_python_package(morpheus-package)
```

Comment:
You need the corresponding call to `morpheus_utils_build_python_package` after all of the calls to `add_subdirectory`. Look at MRC or the MRC quickstart for examples.